lib/Backend/Lower.cpp

```diff
@@ -8197,8 +8200,12 @@ Lowerer::LowerLdElemI(IR::Instr * instr, IR::JnHelperMethod helperMethod, bool i
     {
```

Comment:
You should probably add an assert that the only bailout kind on the instruction is BailOnStackArgsOutOfActualsRange